Orange/widgets/utils/itemmodels.py

```diff
@@ -869,8 +869,19 @@ class DomainModel(VariableListModel):
                  ATTRIBUTES)
     PRIMITIVE = (DiscreteVariable, ContinuousVariable)
 
-    def __init__(self, order=SEPARATED, placeholder=None,
+    def __init__(self, order=SEPARATED, separators=True, placeholder=None,
                  valid_types=None, alphabetical=False, skip_hidden_vars=True, **kwargs):
+        """
+
+        :param order: How attributes, metas and classes are ordered and where between
+                      them are separators.
+        :param separators: Separators are ignored when set to False despite being set in order.
+        :param placeholder: The first element.
+        :param valid_types: For instance continuous, discrete, etc. variables.
+        :param alphabetical: Alphabetical order of attributes or as they are ordered in domain.
+        :param skip_hidden_vars: hidden vars can be hidden or shown
```

Comment:
This is not a description of the option. It also does not explain what are hidden vars.